### PR TITLE
doc: envio indexing documentation

### PR DESCRIPTION
**Pull request by DenhamPreen**
New attempt of this name issue
https://github.com/OffchainLabs/arbitrum-docs/pull/954

### CODE REVIEW

**Comment by DenhamPreen** [comment]
@cla-bot check

**Comment by DenhamPreen** [comment]
I have signed the above form, not sure whats going on.

**Comment by hkalodner** [comment]
@cla-bot check

**Comment by DenhamPreen** [comment]
Thanks @hkalodner 🙏

File: arbitrum-docs/for-devs/third-party-docs/Envio/envio.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+- GraphQL Schema (defaults to `schema.graphql`)
+- Event Handlers (defaults to `src/EventHandlers.*` depending on the language chosen)
+
+These files are auto-generated according to the template and language chosen by running the `envio init` command.
+
+### Contract Import Tutorial
+
+This walkthrough explains how to initialize an indexer using a single or multiple contracts that are already deployed on Arbitrum. This process allows a user to quickly and easily start up a basic indexer and a queryable GraphQL API for their application in less than 3 minutes.
+
+### Intialize your indexer
```
Comment (symbolpunk) [inline arbitrum-docs/for-devs/third-party-docs/Envio/envio.md:53]:
```suggestion
### Initialize your indexer
```

File: arbitrum-docs/for-devs/third-party-docs/Envio/envio.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+## Envio HyperSync
+
+Envio supports [HyperSync](https://docs.envio.dev/docs/hypersync) on Arbitrum.
+
+HyperSync is an accelerated data query layer for the Arbitrum networks, providing APIs that bypass JSON-RPC for 20-100x faster syncing of historical data. HyperSync is used by default in Envio HyperIndex, with the use of RPC being optional. Using HyperSync, application developers do not need to worry about RPC URLs, rate-limiting, or managing infrastructure and can easily sync large datasets in a few minutes, something that would usually take hours or days via RPC.
+
+HyperSync is also available as a standalone API for data analytic use cases. Data analysts can interact with the HyperSync API using JavaScript, Python, or Rust clients and extract data in JSON, Arrow, or Parquet formats. For more information, visit the HyperSync documentation [here](https://docs.envio.dev/docs/overview-hypersync).
+
+## HyperIndex Key Features
```
Comment (symbolpunk) [inline arbitrum-docs/for-devs/third-party-docs/Envio/envio.md:27]:
```suggestion
## HyperIndex key features
```

File: arbitrum-docs/for-devs/third-party-docs/Envio/envio.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+- Contract Import: Autogenerate the key boilerplate for an entire Indexer project off a single or multiple smart contracts. Deploy within minutes.
+
+- Multi-chain Support: Aggregate data across multiple networks into a single database. Query all your data with a unified GraphQL API.
+
+- Asynchronous Mode: Fetch data from off-chain storage such as IPFS, or contract state (e.g. smart contract view functions).
+
+- Quickstart Templates: Use pre-defined indexing logic for popular OpenZeppelin contracts (e.g. ERC-20).
+
+## Getting Started
```
Comment (symbolpunk) [inline arbitrum-docs/for-devs/third-party-docs/Envio/envio.md:37]:
```suggestion
## Getting started
```

File: arbitrum-docs/for-devs/third-party-docs/Envio/envio.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+The following files are required to run the Envio indexer:
+
+- Configuration (defaults to `config.yaml`)
+- GraphQL Schema (defaults to `schema.graphql`)
+- Event Handlers (defaults to `src/EventHandlers.*` depending on the language chosen)
+
+These files are auto-generated according to the template and language chosen by running the `envio init` command.
+
+### Contract Import Tutorial
```
Comment (symbolpunk) [inline arbitrum-docs/for-devs/third-party-docs/Envio/envio.md:49]:
```suggestion
### Contract import tutorial
```

File: arbitrum-docs/for-devs/third-party-docs/Envio/envio.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+  goerli
+> arbitrum-one
+  arbitrum-nova
+  bsc
+  gnosis
+v polygon
+[↑↓ to move, enter to select, type to filter]
+```
+
+#### Enter in the address of the contract to import
```
Comment (symbolpunk) [inline arbitrum-docs/for-devs/third-party-docs/Envio/envio.md:108]:
```suggestion
#### Enter the address of the contract to import
```

File: arbitrum-docs/for-devs/third-party-docs/Envio/envio.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+  Add a new network for same contract
+  Add a new contract (with a different ABI)
+[Current contract: BribeVotingReward, on network: arbitrum-one]
+```
+
+The `Contract Import` process will prompt the user whether they would like to finish the import process or continue adding more addresses for same contract on same network, addresses for same contract on different network or a different contract.
+
+For more information on contract import feature, visit the documentation [here](https://docs.envio.dev/docs/contract-import).
+
+## Envio Examples
```
Comment (symbolpunk) [inline arbitrum-docs/for-devs/third-party-docs/Envio/envio.md:143]:
```suggestion
## Envio examples
```

File: arbitrum-docs/for-devs/third-party-docs/Envio/envio.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+
+Click [here](https://docs.envio.dev/docs/example-uniswap-v3) for Envio HyperIndex examples.
+Click [here](https://docs.envio.dev/docs/hypersync-clients) for Envio HyperSync examples.
+
+## Getting Help
```
Comment (symbolpunk) [inline arbitrum-docs/for-devs/third-party-docs/Envio/envio.md:148]:
```suggestion
## Getting help
```

**Review by symbolpunk — APPROVED** [review]
LGTM - thank you for following the contribution guidelines 🙏

**Pull request merged**